conf/augmentations.py

```diff
@@ -0,0 +1,84 @@
+from hydra.core.config_store import ConfigStore
```

Comment:
I think your problem is that you are not importing this module so the configs never gets registered.

Reply:
Fixed it!